miniapp/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,8 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
@corycaywood @ts-nitish-a-jain if we let the hostapp in full control of permission, should we define those permissions on hostapp `AndroidManifest`?

Reply:
For sure.